src/main/java/core/selenium/WebElementAction.java

```diff
@@ -157,4 +157,15 @@ public boolean isElementPresent(final By by, final int intervalTime) {
                     .getWebDriverConfig().getImplicitWaitTime(), TimeUnit.MILLISECONDS);
         }
     }
+
+    /**
+     * Clicks the checkbox elements.
+     * @param webElement checkbox.
```

Comment:
You should add a newline between the title and the params of the method.

Reply:
ok